src/python/turicreate/toolkits/_model.py

```diff
@@ -68,7 +68,7 @@ def load_model(location):
         dir_archive_exists = file_util.exists(os.path.join(model_path, 'dir_archive.ini'))
     else:
         model_path = location
-        if protocol in ['http', 'https']:
+        if protocol in ['http', 'https', 's3']:
```

Comment:
Is there an easy way to test this? With boto maybe?

Reply:
Sadly, no. Right now we don't have any S3 unit test.